package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "check": "npm run test && npm run lint && npm run prettier",
     "dev": "next dev -p 3000",
-    "build": "next build && npm run parse-primaryCodePath",
+    "build": "npm run parse-primaryCodePath && npm run build:names && next build",
```

Comment:
Should we actually run build:names as part of a basic build? Seems like since the names file is checked in, we probably want to just leave as is except in exceptional circumstances.

Reply:
Yeah that sounds right actually. I wonder if I should remove both that and the primary code path one since both results of those scripts are already checked into the repo?

Reply:
@sarahmcdougall Do we def need the primary code path parsing to happen in the build script or is it just there for convenience?

Reply:
@mgramigna nah we just decided to put it in there for convenience. It doesn't need to happen in the build script